Gemfile

```diff
@@ -38,6 +38,8 @@ gem 'deacon', '~> 1.0'
 gem 'webpack-rails', '~> 0.9.8'
 gem 'mail', '~> 2.6'
 gem 'nokogiri', '< 1.7' if RUBY_VERSION.start_with? '2.0.'
+gem 'dynflow', '>= 0.8.20', '< 1.0.0'
+gem 'daemons'
```

Comment:
if you plan on running `bin/foreman-tasks`, you will need this gem. If it's not installed, the system will tell you that this gem is missing.
Originally, it was in a separate group, so the user could decide if it's wanted or no.